ui/main/src/app/modules/settings/components/settings/settings.component.ts

```
@@ -140,6 +140,13 @@ export class SettingsComponent implements OnInit, OnDestroy {
         });
     }
 
+    private numericalInputChecker(inputElement: HTMLInputElement) {
+        const numericValue = parseFloat(inputElement.value);
+        if (isNaN(numericValue) || numericValue < 0) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)